Header                          header
string                          category
uint16                          bbox_xmin
uint16                          bbox_xmax
uint16                          bbox_ymin
uint16                          bbox_ymax
float64                         probability
geometry_msgs/TransformStamped  trans
geometry_msgs/PoseStamped       pose
